db/migrate/20170127094357_add_caching_enabled_to_compute_resource.rb

```diff
@@ -0,0 +1,5 @@
+class AddCachingEnabledToComputeResource < ActiveRecord::Migration
+  def change
+    add_column :compute_resources, :caching_enabled, :boolean, default: true
```

Comment:
@tbrisker : Yes, it will. But it's not shown in the UI or API. So it figured, it doesn't really matter.   Do you see that differently?